### ISSUE TITLE
channels.createChannel with megagroup true returns INPUT_FETCH_ERROR

### ISSUE THREAD

**Issue by pokearaujo** [alik0211/mtproto-core#106]
Hi,

When i call channels.createChannel with megagroup = true returns a 400 error

```

    const group = await this.call('channels.createChannel', {
      megagroup: true,
      title: 'Test group',
      about: 'Test nice group',
    });

```

Returns:

```
channels.createChannel error: {
  _: 'mt_rpc_error',
  error_code: 400,
  error_message: 'INPUT_FETCH_ERROR'
}
```

Used with latest version lib in npm.
Thanks;

**Comment by pokearaujo** [comment]
Removing this from builder in channels.createChannel
```
flag(Bool, params.broadcast);
flag(Bool, params.megagroup);
```

He works normally, but i don't know if is right.

Thanks again;

**Comment by alik0211** [comment]
Good point!

This issue is fixed in version 5.0.0 - https://t.me/mtproto_core/23
